recurser/string-is
# Add a share feature

Replace this with a detailed explanation of the change


## How to test the PR

Load an example, click the `Share` button, and open the URL in a new incognito window. The input and settings should be copied.
